modules/margin/src/main/java/com/opengamma/sdk/margin/InvokerMarginClient.java

```diff
@@ -71,6 +72,7 @@ public CcpsResult listCcps() {
         .url(invoker.getServiceUrl().resolve("/margin/v1/ccps"))
         .get()
         .header("Accept", MEDIA_JSON.toString())
+        .header("User-Agent", generateUserAgent())
```

Comment:
Rather than adding it manually to each request, you should use an interceptor, see `ServiceInvoker`.